meetstar_site/mainpage/views.py

```diff
@@ -35,10 +36,13 @@ def randomize(request):
     event_id = request.GET['event_id']
     try:
         event = Events.objects.get(id=event_id)
-        event.randomize()
+        winner = event.randomize()
     except Events.DoesNotExist:
         return HttpResponse('Event with ID {0} doesnt exist'.format(event_id))
-
+    all_users = UsersInEvent.objects.filter(id=event_id)
+    for user in all_users:
+        send_mail('It is time for event', 'The winner is',
```

Comment:
Next thing to do - use pretty template to make it human readable. You can use htmlemail.io as a source